### ISSUE TITLE
(2.2.11) Console Not Functioning how it should?

### ISSUE THREAD

**Issue by ThomasHallGit** [SRB2-Mobile/SRB2-Android#46]
So I'm probably not the only one who noticed this, but something is wrong with the console. Or more specifically, typing commands in.

Whenever I try to type in a command, the word(s) for some reason gets deleted after tapping the space key once, meaning I'll have to type it in again. If I launch an unknown command and do it again, it works as it should, but if you close the console and open it again, the bug happens again. It's not game-breaking, but it's annoying when using mods with console commands such as BuddyEX by SMS Alfredo for example.

**Comment by xsm2** [comment]
You are not the only person with the console problem, it has happenned to me too and I know that more than one person in the #android-help discord channel had the problem. 

I have never searched anything in the addons menu so I haven't tested that.

**Comment by bitten2up** [comment]
yeah i have seen it, i mean you could use console scripts, but that could become inconvenience
